### DISCUSSION TITLE
Link error cross referencing a figure in book pdf

### DISCUSSION THREAD

**Discussion by fredguth** [quarto-dev/quarto-cli#3839] [Q&A]
MWE: https://github.com/fredguth/quarto-mwe

Created a figure in `intro.qmd` chapter that is referenced in `summary.qmd`. `Html` references fine, but `pdf` link to figure is `??`  not `1.1` as expected. Despite rendering `??`, the link redirects to the right place.

 
<img width="922" alt="Screenshot 2023-01-03 at 17 17 52" src="https://user-images.githubusercontent.com/392879/210434773-891bbae9-5ea7-46ac-8954-55d75bfbed57.png">

**Comment by rgaiacs** [comment]
In https://github.com/fredguth/quarto-mwe/blob/main/intro.qmd, can you try

``````
::: {.content-hidden unless-format="pdf"}
{{< pagebreak >}} 
\vspace{2cm}

::: {.column-body}

![The Blind Men and the Elephant.](/Images/elephant.pdf){#fig-elephant}

:::

:::
``````

instead of

``````
::: {.content-hidden unless-format="pdf"}
{{< pagebreak >}} 
\vspace{2cm}

::: {.column-body #fig-elephant}

![](/Images/elephant.pdf)

The Blind Men and the Elephant.

:::

:::
``````

**Reply by fredguth** [reply]
This works.  But should I mark it as solved?  Because according to the documentation, shouldn't it work the way it was earlier?

By the way, I am using the tufte-latex document class and for some weird reason it does not work when the generated tex puts the label inside the caption (which is a totally normal thing to do).  I will have to find out how to fix that in tufte-class.
